.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,73 @@
+name: Bug report
+description: File a bug report for chrome-devtools-mcp
+title: '[Bug] '
```

Comment:
Should we prefix bugs with a title? This is redundant to labels

Reply:
Speaking of, can we add the `bug` label here?

Reply:
Done